### DISCUSSION TITLE
Are camera and zone count detection MQTT messages retained?

### DISCUSSION THREAD

**Discussion by kiat-huang** [blakeblackshear/frigate#1993] [General]
I'd like them to be so that configured sensors in HA will always know the current count.  The docs didn't say which messages published are with the retain flag and I looked at the MQTT code and (I'm no developer) couldn't spot where the retain flags are decided. Am I missing something or this a feature that would need to be requested?  (a setting in the config file or whatever).

**Comment by blakeblackshear** [comment]
Only the snapshots are published with the retain flag. It would need to be requested.
